cargo-apk/src/apk.rs

```diff
@@ -225,13 +225,41 @@ impl<'a> ApkBuilder<'a> {
 
         let signing_key = self.manifest.signing.get(profile_name);
 
-        let signing_key = match (signing_key, is_debug_profile) {
-            (Some(signing), _) => Key {
+        let signing_key = if let Some(signing) = signing_key {
+            Key {
                 path: crate_path.join(&signing.path),
                 password: signing.keystore_password.clone(),
-            },
-            (None, true) => self.ndk.debug_key()?,
-            (None, false) => return Err(Error::MissingReleaseKey(profile_name.to_owned())),
+            }
+        } else {
+            let env_profile_name = profile_name.to_uppercase().replace('-', "_");
+
+            let path = {
+                let profile_env = format!("CARGO_APK_{}_KEYSTORE", env_profile_name);
+                std::env::var_os(&profile_env).map(PathBuf::from)
+            };
+
+            let password = {
+                let profile_env = format!(
+                    "CARGO_APK_{}_KEYSTORE_PASSWORD",
+                    profile_name.to_uppercase().replace('-', "_")
+                );
+
+                std::env::var(&profile_env).ok()
+            };
+
+            if is_debug_profile {
+                if let (Some(path), Some(password)) = (path, password) {
+                    Key { path, password }
+                } else {
+                    self.ndk.debug_key()?
+                }
+            } else {
+                Key {
+                    path: path.ok_or_else(|| Error::MissingReleaseKey(profile_name.to_owned()))?,
+                    password: password
+                        .ok_or_else(|| Error::MissingReleaseKey(profile_name.to_owned()))?,
+                }
+            }
         };
 
         Ok(apk.add_pending_libs_and_align()?.sign(signing_key)?)
```

Comment:
How do you feel about using a little `println!("Signing APK with {}", signing_key.path);` here, just to at least help the user validate what is going on, in case they don't get the environment variables right?